java/ql/src/Security/CWE/CWE-078/ExecTaintedEnvironment.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Building a command with an injected environment variable
+ * @description Using externally controlled strings in the environment variables
+ *              passed to a command line is vulnerable to malicious changes to the
+ *              environment of a subprocess.
```

Comment:
The following is a more concise version, does it contain all the information still?

```suggestion
 * @description Passing environment variables containing externally controlled 
 * strings to a command line is vulnerable to malicious changes to the
 *              environment of a subprocess.
```